src/convert.ts

```diff
@@ -640,6 +642,7 @@ const convertSpells = (ddbCharacter: DdbCharacter): AlchemySpell[] => {
   return Object.entries(ddbCharacter.spells)
     .filter(([origin, spells]) => origin !== "item" && spells)
     .flatMap(([_origin, spells]) => spells)
+    .filter(spell => spell.definition)
```

Comment:
this is smart; I think we can also remove the corresponding `const spell = ddbSpell.definition` on L651

Reply:
I _think_ we still need that one. This is just making sure that we don't pass any spells with a nullish `definition` field down to the conversion function.

Reply:
oh I'm dumb, this is `filter` not `map`. good catch